Makefile.d/sync-external-ip.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -eu -o pipefail
+
+for node in $(kubectl get nodes -o name); do
+	# Set ExternalIP
+	host_ip="$(kubectl get "${node}" -o jsonpath='{.metadata.labels.usernetes/host-ip}')"
+	kubectl patch "${node}" --type=merge --subresource status --patch \
+		"\"status\": {\"addresses\": [{\"type\":\"ExternalIP\", \"address\": \"${host_ip}\"}]}"
+
+	# Propagate ExternalIP to flannel
+	# https://github.com/flannel-io/flannel/blob/v0.24.4/Documentation/kubernetes.md#annotations
+	kubectl annotate "${node}" flannel.alpha.coreos.com/public-ip-overwrite=${host_ip}
```

Comment:
(On worker nodes, flanneld fails before fetching this annotation from the api server)
```
2024-03-31T20:12:47.312421896Z stderr F I0331 20:12:47.312181       1 main.go:209] CLI flags config: {etcdEndpoints:http://127.0.0.1:4001,http://127.0.0.1:2379 etcdPrefix:/coreos.com/network etcdKeyfile: etcdCertfile: etcdCAFile: etcdUsername: etcdPassword: version:false kubeSubnetMgr:true kubeApiUrl: kubeAnnotationPrefix:flannel.alpha.coreos.com kubeConfigFile: iface:[] ifaceRegex:[] ipMasq:true ifaceCanReach: subnetFile:/run/flannel/subnet.env publicIP: publicIPv6: subnetLeaseRenewMargin:60 healthzIP:0.0.0.0 healthzPort:0 iptablesResyncSeconds:5 iptablesForwardRules:true netConfPath:/etc/kube-flannel/net-conf.json setNodeNetworkUnavailable:true}
2024-03-31T20:12:47.312494768Z stderr F W0331 20:12:47.312271       1 client_config.go:618] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
2024-03-31T20:13:17.343115429Z stderr F E0331 20:13:17.342306       1 main.go:226] Failed to create SubnetManager: error retrieving pod spec for 'kube-flannel/kube-flannel-ds-7l6rt': Get "https://10.96.0.1:443/api/v1/namespaces/kube-flannel/pods/kube-flannel-ds-7l6rt": dial tcp 10.96.0.1:443: i/o timeout
```